curriculum/challenges/english/24-b1-english-for-developers/learn-present-perfect-while-talking-about-accessibility/672deeb8a3167fda01889d18.md

```diff
@@ -0,0 +1,104 @@
+---
+id: 672deeb8a3167fda01889d18
+title: Task 106
+challengeType: 22
+dashedName: task-106
+---
+
+<!-- (Audio) James: Yes, the interactive activities haven't been as effective as we thought. It seems that some of the scenarios weren't relevant enough for the participants. -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Yes, the BLANK BLANK haven't been as effective as we thought. It seems that some of the BLANK weren't relevant enough for the participants.`
+
+## --blanks--
+
+`interactive`
+
+### --feedback--
+
+This word describes something that involves active participation, like exercises or activities.
+
+---
+
+`activities`
+
+### --feedback--
+
+This word refers to tasks or exercises that people take part in. It's plural form.
+
+---
+
+`scenarios`
+
+### --feedback--
+
+This word means examples or situations created to help people understand or practice something. It's plural form.
+
+# --explanation--
+
+`Interactive` means something that lets people participate or take action, rather than just watching or listening. For example,
+
+- `The training included interactive games to help learners practice their skills.` - It shows that people are actively involved in the games, not just observing.
+
+`Activity` is a task or an exercise people do to learn or practice skills. For instance,
+
+- `The activities in the workshop helped everyone understand the material better.` - This means that the tasks helped people engage with and understand the content.
+
+`Scenario` is a specific situation or example created to help people imagine or explore real-life situations. For example,
+
+- `In the project meeting, we discussed different scenarios to prepare for possible challenges.` - Here, it means to imagine situations that help the team think about potential issues and find solutions ahead of time.
+
+# --scene--
+
+```json
+{
+  "setup": {
+    "background": "interview-room1.png",
+    "characters": [
+      {
+        "character": "James",
+        "position": {
+          "x": 50,
+          "y": 15,
+          "z": 1.2
+        },
+        "opacity": 0
+      }
+    ],
+    "audio": {
+      "filename": "B1_4-4.mp3",
+      "startTime": 1,
+      "startTimestamp": 27.7,
+      "finishTimestamp": 36.86
+    }
+  },
+  "commands": [
+    {
+      "character": "James",
+      "opacity": 1,
+      "startTime": 0
+    },
+    {
+      "character": "James",
+      "startTime": 1,
+      "finishTime": 10.16,
+      "dialogue": {
+        "text": "Yes, the interactive activities haven't been as effective as we thought. It seems that some of the scenarios weren't relevant enough for the participants.",
+        "align": "center"
+      }
+    },
+    {
+      "character": "James",
+      "opacity": 0,
+      "startTime": 10.66
+    }
+  ]
+}
+```
```

Comment:
```suggestion
# --scene--

```json
{
  "setup": {
    "background": "interview-room1.png",
    "characters": [
      {
        "character": "James",
        "position": {
          "x": 50,
          "y": 15,
          "z": 1.2
        },
        "opacity": 0
      }
    ],
    "audio": {
      "filename": "B1_4-4.mp3",
      "startTime": 1,
      "startTimestamp": 27.7,
      "finishTimestamp": 36.96
    }
  },
  "commands": [
    {
      "character": "James",
      "opacity": 1,
      "startTime": 0
    },
    {
      "character": "James",
      "startTime": 1,
      "finishTime": 5.8,
      "dialogue": {
        "text": "Yes, the interactive activities haven't been as effective as we thought.",
        "align": "center"
      }
    },
    {
      "character": "James",
      "startTime": 6.28,
      "finishTime": 10.26,
      "dialogue": {
        "text": "It seems that some of the scenarios weren't relevant enough for the participants.",
        "align": "center"
      }
    },
    {
      "character": "James",
      "opacity": 0,
      "startTime": 10.76
    }
  ]
}
```
```